test/unstableAnonScript/specialCases/SKIPIF

```diff
@@ -0,0 +1 @@
+../SKIPIF
```

Comment:
I think you could get away with putting a skipif on the directory itself, e.g. `test/unstableAnonScript.skipif` rather than having the directory-wide skipif.  That would avoid the need to have one in the subdirectory as well, I'm pretty sure

Reply:
Oh cool! I'll do that instead.

Reply:
I tried doing that and ran a `(paratest) -dirs unstableAnonScript` and it didn't skip the tests in the subdirectory. It could be because now the skipif is actually outside the directory being tested and the `-dirs` flag confuses it? It might be worth filing an issue. 
A full paratest skips those tests fine so I guess it should be fine for resolving the nightly failure itself.

Reply:
I know there's a flag "--respect-skipifs" for start_test but I can't remember if that works with paratest . . .